lib/teselagen/api/client.py

```diff
@@ -343,14 +343,19 @@ def select_laboratory(self, lab_name: Optional[str]=None, lab_id: Optional[int]=
 
         Args:
             lab_name (str): The name of the lab. If not set, the method will
-                use the lab_id parameter
+                use the lab_id parameter. If both parameters are ommited, Lab
+                is set to "Common".
             lab_id (int): ID of the lab. If not set the method will use the
                 lab_name parameter as lab identifier
         """
         identifier = lab_name if lab_id is None else str(lab_id)
         search_field = 'name' if lab_id is None else 'id'
         if identifier is None:
-            raise ValueError("Received None lab identifiers")
```

Comment:
Does it makes sense to include an argument flag (such as `strict: bool = True`) to raise an exception when `identifier is None`? In order to avoid cases when a user may be inadvertently uploading sensitive data to common lab. If `strict == False`, then `"common"` lab is selected when `identifier is None`.

Reply:
Yes, you are right. Maybe just raise an error always when the identifier is None.

Reply:
Done!